tests/unit/test_endpoint.py

```diff
@@ -1,4 +1,4 @@
-# Copyright 2012-2014 Amazon.com, Inc. or its affiliates. All Rights Reserved.
+# Copyright 2012-2015 Amazon.com, Inc. or its affiliates. All Rights Reserved.
```

Comment:
It doesn't look like there are any other changes to this file. Any reason to update the copyright year?